binary_search_tree/binary_search_tree.py

```diff
@@ -1,14 +1,35 @@
 class BinarySearchTree:
-  def __init__(self, value):
-    self.value = value
-    self.left = None
-    self.right = None
+    def __init__(self, value):
+        self.value = value
+        self.left = None
+        self.right = None
 
-  def insert(self, value):
-    pass
+    def insert(self, value):
+        if value >= self.value:
```

Comment:
Interesting that you used >= here. I never considered what would happen if you tried in insert a duplicate node.